src/03_modules.py

```diff
@@ -10,22 +10,23 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
-
+print(sys.argv)
```

Comment:
Because of the "one per line" instruction, it's better to loop through the args like:
`for arg in sys.argv:
    print(arg)`

Reply:
Thankyou, I assumed because there was only one argument at the moment I didn't have to loop